src/components/dev-hub/section-header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { colorMap, fontSize, screenSize } from './theme';
+import { P } from './text';
+
+const SectionHeader = styled('div')`
+    border-bottom: 1px solid ${colorMap.greyDarkOne};
+    font-size: ${fontSize.small};
+    letter-spacing: 2px;
+    @media ${screenSize.upToMedium} {
+        display: block;
+    }
+    text-transform: uppercase;
+`;
+
+export default ({ text }) => (
```

Comment:
Here is a cool trick! If we use a prop named `children`, then when we use this component we can do something like this:

```
<SectionHeader>For Academia</SectionHeader>
```

instead of

```
<SectionHeader text="For Academia" />
```

Personally I think the first one is more natural with html, but I'll leave it to you :)

Reply:
The first one definitely feels more natural. Thanks!